content/tutorials/provision-dashboards-and-data-sources.md

```diff
@@ -113,15 +116,16 @@ providers:
     options:
       path: /var/lib/grafana/dashboards # The path to the dashboard definitions
 ```
+((I do not have this path in my Grafana installation on my Windows machine. This is my path: C:\Program Files\GrafanaLabs\grafana\public\dashboards))
```

Comment:
Should we change this to:

```
options:
  path: <path to dashboard definitions>
```

Reply:
We should probably include something about the format, because obviously just copying and pasting the Windows path does not work.

It did work when I changed it to C:/Program Files/GrafanaLabs/grafana/public/dashboards

Btw, this is a vanilla Windows installation with default settings, so I think we could safely include this path as an option.